packages/block-editor/src/components/inserter/test/block-types-tab.js

```diff
@@ -138,16 +135,14 @@ describe( 'InserterMenu', () => {
 		expect( blocks[ 2 ].textContent ).toBe( 'Some Other Block' );
 	} );
 
-	it( 'displays child blocks UI when root block has child blocks', () => {
-		useSelect.mockImplementation( () => true );
-
+	it( "doesn't display child blocks UI", () => {
```

Comment:
If we delete the `ChildBlocks` component, then I think we can also remove this test as well.